doc/INSTALL.md

```diff
@@ -61,6 +61,10 @@ For development or running tests, get additional dependencies:
     sudo apt-get install -y valgrind libpq-dev shellcheck cppcheck \
       libsecp256k1-dev jq
 
+If you want to build the Rust plugins (currently, cln-grpc):
+
+	sudo apt-get install -y cargo rustfmt
```

Comment:
not sure if this will install also the rust compiler `rustc`, maybe we can reproduce the official installer guide in rust? https://www.rust-lang.org/tools/install
```suggestion
	curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
	
Can you see the official process [on rust official site for other types of installation](https://www.rust-lang.org/tools/install)
```

Maybe the last line need a rework

Reply:
Yes, it does, I tested on a clean install.